specification/applicationinsights/ApplicationInsights.LiveMetrics/models.tsp

```diff
@@ -0,0 +1,567 @@
+  @header("x-ms-qps-subscribed")
+  subscribed: string;
+
+  @doc("An encoded string that indicates whether the collection configuration is changed.")
+  @header("x-ms-qps-configuration-etag")
+  configurationEtag: string;
+
+  @doc("Recommended time (in milliseconds) before an SDK should ping the service again. This header exists only when instrumentation key is not watched by UX.")
+  @header("x-ms-qps-service-polling-interval-hint")
+  pollingInterval?: string;
+
+  @doc("Contains a URI of the service endpoint that an SDK must permanently use for the particular resource. This header exists only when SDK is talking to QuickPulse's global endpoint.")
+  @header("x-ms-qps-service-endpoint-redirect-v2")
+  redirectEndpoint?: string;
+
+  @doc("Corresponding status code.")
+  @statusCode
+  _: 200;
+}
+
+/**
+ * QuickPulse /post request parameters, either in query string, or in request body
+ */
+model PostRequest {
+  ...PostRequestQueryStrings;
+  ...PostRequestHeaders;
+  ...PostRequestBody;
+}
+
+/**
+ * QuickPulse post request query strings
+ */
+model PostRequestQueryStrings {
+  @doc("The ikey of the target Application Insights component for which client sdk calls /post.")
+  @query
+  ikey: string;
+
+  @doc("Deprecated. An alternative way to pass api key. Use AAD auth instead.")
+  @query
+  apiKey?: string;
+}
+
+/**
+ * QuickPulse ping response headers
+ */
+model PostRequestHeaders {
+  @doc("An encoded string that indicates whether the collection configuration is changed.")
+  @header("x-ms-qps-configuration-etag")
+  configurationEtag?: string;
+
+  @doc("Timestamp when SDK transmits the metrics and documents to QuickPulse. A 8-byte long type of ticks.")
+  @header("x-ms-qps-transmission-time")
+  transmissionTime?: int64;
+}
+
+/**
+ * QuickPulse ping request body
+ */
+#suppress "@azure-tools/typespec-azure-core/request-body-problem" "Need to follow the casing style of the existing SDK contract."
+model PostRequestBody {
+  @doc("Data contract between SDK and QuickPulse. /QuickPulseService.svc/ping uses this as a backup source of machine name, instance name and invariant version.")
+  @body
+  monitoringDataPoints?: MonitoringDataPoint[];
+}
+
+/**
+ * Represents the headers for the QuickPulse service.
+ */
+model PostResponseHeaders {
+  @doc("Tells SDK whether the input instrumentation key is subscribed to by UX.")
+  @header("x-ms-qps-subscribed")
+  subscribed: string;
+
+  @doc("An encoded string that indicates whether the collection configuration is changed.")
+  @header("x-ms-qps-configuration-etag")
+  configurationEtag: string;
+}
+
+/**
+ * Monitoring data point coming from SDK, which includes metrics, documents and other metadata info.
+ */
+#suppress "@azure-tools/typespec-azure-core/casing-style" "Need to follow the casing style of the existing SDK contract."
+model MonitoringDataPoint {
+  @doc("AI SDK version.")
+  Version: string;
```

Comment:
I take it your existing API already uses PascalCase? We prefer camelCase, but if this is what you support now it's understandable.